client-javax/build.gradle

```diff
@@ -25,7 +25,7 @@ dependencies {
     implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.17.0'
     implementation 'org.glassfish.jersey.core:jersey-client:2.36'
     implementation 'org.glassfish.jersey.media:jersey-media-json-jackson:2.36'
-    implementation 'org.glassfish.jersey.media:jersey-media-multipart:2.36'
+    implementation 'org.glassfish.jersey.media:jersey-media-multipart:3.1.6'
```

Comment:
This change updates `jakarta.ws.rs:jakarta.ws.rs-api:3.1.0` which uses `jakarta` namespace. This won't work because it is a `javax` client which suppose to use `javax` namespace instead. It's not clear why this change was introduced by dependabot since we have [rule](https://github.com/DataBiosphere/terra-billing-profile-manager/blob/bc5e97852e6b65038b88c3e80752c0967716bdbd/.github/dependabot.yml#L56) for ignoring such dependency updates for `org.glassfish.jersey* libs. Maybe it is because old version doesn't have patch component.  Anyway it should be closed.